docs/content/direct/architecture.md

```diff
@@ -263,166 +327,98 @@ handler and the work queue as follows:
     - Creates GVR key
     - Registers Event Handler
     - Starts Informer
-    - Indexes informer and lister in a map by GVR key
+    - Stores informer and lister in a map indexed by GVR
 - Waits for all caches to sync
 - Starts N workers to process work queue
 
```

Comment:
Missing from docs: at startup the binding controller currently notes all binding-policies.

Reply:
Revised.